kura/examples/org.eclipse.kura.example.eddystone.advertiser/OSGI-INF/EddystoneAdvertiser.xml

```diff
@@ -1,12 +1,16 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!--
 
-    Copyright (c) 2017 Eurotech and/or its affiliates and others
+   Copyright (c) 2017 Eurotech, 2020 Eurotech and/or its affiliates and others
```

Comment:
not aligned with the others

Reply:
Don't know. It displays ok with my editor.